automol/vmat.py

```diff
@@ -181,13 +183,13 @@ def coordinate_key_matrix(vma, shift=0):
             (atm_key,) + key_row[: col + 1] if key_row[col] is not None else None
             for col in range(3)
         ]
-        for atm_key, key_row in zip(atm_keys, key_mat)
+        for atm_key, key_row in zip(atm_keys, key_mat, strict=False)
     ]
 
     return tuple(map(tuple, coo_key_mat))
 
 
-def coordinates(vma, shift=0, multi=True):
+def coordinates(vma: VMatrix, shift=0, multi=True):
```

Comment:
With the above `CoordinateKey` type defined, the return value here can be given as `-> dict[Name, CoordinateKey]`.